core/src/Cabana_CommunicationPlan.hpp

```diff
@@ -59,6 +59,13 @@ struct CountSendsAndCreateSteeringAlgorithm<Kokkos::Experimental::HIP>
     using type = CountSendsAndCreateSteeringAtomic;
 };
 #endif // end KOKKOS_ENABLE_HIP
+#ifdef KOKKOS_ENABLE_OPENMPTARGET
+template <>
+struct CountSendsAndCreateSteeringAlgorithm<Kokkos::Experimental::OpenMPTarget>
```

Comment:
No duplication because ostensibly this is for a GPU which should have good atomics?

Reply:
That was my assumption, but worth discussing